gateway/mw_oas_validate_request.go

```diff
@@ -60,12 +61,27 @@ func (k *ValidateRequest) ProcessRequest(w http.ResponseWriter, r *http.Request,
 
 	router, err := gorillamux.NewRouter(&oasSpec)
 	if err != nil {
-		return fmt.Errorf("request validation error: %v", err), http.StatusBadRequest
+		return nil, http.StatusOK
```

Comment:
Err is left unhandled, not logged or returned. Fix other similar examples.

Reply:
The idea here is that we doesn't want to return error here (see other middlewares). When we return it goes to the next middleware. Why isn't it logging here? Because, this is a temporary code and it will be moved to a common place where we call it one time. That's why I didn't add a log statement. Also, we already check whether the OAS is valid one while saving it. An invalid OAS cannot come until this point. However, I will add log statement anyway when we move it to a common place to stop running it on every request.